src/components/leaderboards/TeamLeaderboard/index.js

```diff
@@ -0,0 +1,143 @@
+"use strict";
+
+var _                       = require('lodash');
+var React                   = require('react/addons');
+var I18nMixin               = require('../../mixins/I18n');
+var DOMInfoMixin            = require('../../mixins/DOMInfo');
+var LeaderboardMixin        = require('../../mixins/Leaderboard');
+var Icon                    = require('../../helpers/Icon');
+var LeaderboardItem         = require('../LeaderboardItem');
+var TeamLeaderboardItem     = require('../TeamLeaderboardItem');
+var numeral                 = require('numeral');
+var addEventListener        = require('../../../lib/addEventListener');
+var removeEventListener     = require('../../../lib/removeEventListener');
+var ReactCSSTransitionGroup = React.addons.CSSTransitionGroup;
+
+module.exports = React.createClass({
+  mixins: [I18nMixin, DOMInfoMixin, LeaderboardMixin],
+  displayName: "TeamLeaderboard",
+  propTypes: {
+    campaignSlug: React.PropTypes.string,
+    campaignUid: React.PropTypes.string,
+    charitySlug: React.PropTypes.string,
+    charityUid: React.PropTypes.string,
+    country: React.PropTypes.oneOf(['au', 'nz', 'uk', 'us']),
+    type: React.PropTypes.oneOf(['team']),
+    limit: React.PropTypes.number,
+    pageSize: React.PropTypes.number,
+    backgroundColor: React.PropTypes.string,
+    textColor: React.PropTypes.string,
+    childWidth: React.PropTypes.number,
+    currencyFormat: React.PropTypes.string,
+    i18n: React.PropTypes.object
+  },
+
+  getDefaultProps: function() {
+    return {
+      type: 'team',
+      limit: 48,
+      pageSize: 12,
+      backgroundColor: null,
+      textColor: null,
+      childWidth: 250,
+      altTemplate: false,
+      currencyFormat: '0[.]00 a',
+      defaultI18n: {
+        raisedTitle: 'Raised',
+        membersTitle: 'Members',
+        symbol: '$',
+        heading: 'Top Teams'
+      }
+    };
+  },
+
+  getInitialState: function() {
+    return {
+      isLoading: false,
+      boardData: [],
+      currentPage: 1,
+      childWidth: '',
+    };
+  },
+
+  componentDidMount: function() {
+    addEventListener(window, 'resize', this.setChildWidth);
+  },
```

Comment:
Fire this.setChildWidth() after you've attached the listener. Otherwise if they don't resize, the childwidth won't be calculated.